product_docs/docs/pgd/4/harp/index.mdx

```diff
@@ -242,3 +242,53 @@ location: dcb
 This applies to any HARP Proxy nodes that are designated in those respective
 data centers as well.
 
+### Health check
```

Comment:
move this section under [Configuration](https://www.enterprisedb.com/docs/pgd/4/harp/04_configuration/#standard-configuration). Before Runtime directives.